README.md

```diff
@@ -47,6 +45,9 @@ Run `./flush-memcache.sh` to flush the development cache.
 
 The build process is automated using `gulp` and configured with `gulpfile.js`.
 
+Verify gulp is installed with `gulp --version`. If it's not installed, run
```

Comment:
Why is the `npm install --save-dev ...` step required? Gulp is already present in `package.json`'s `devDependencies`.